### ISSUE TITLE
Menu Item does not highlight when selected

### ISSUE THREAD

**Issue by darklord1990** [yaminncco/vue-sidebar-menu#49]
As you can see, Menu "ALL" has highlight after clicked but menu "product1" does not.
![image](https://user-images.githubusercontent.com/30645479/57125155-e3850100-6dba-11e9-8bf6-3bf8c423e2cb.png)

![image](https://user-images.githubusercontent.com/30645479/57125266-5f7f4900-6dbb-11e9-9e9f-fc75423a40d5.png)

Here is the code of menu :

```
    {
        href: '/domain',
        title: 'All',
        icon: 'fa fa-file-alt'
    },
    {
        href: '/domain/?pid=1',
        title: 'product1',
        icon: 'fa fa-file-alt'
    },
    {
        href: '/domain/?pid=2',
        title: 'product2',
        icon: 'fa fa-file-alt'
    }
```

And the route :
```
    {
        path: '/domain',
        component: require('./pages/Domain.vue').default,
        meta: {
            requiredUser: ['default']
        }
    },
    {
        path: '/domain/?pid=*',
        component: require('./pages/Dashbroad.vue').default,
        meta: {
            requiredUser: ['default']
        }
    }
```

how can I fix it ?  Please help

Thanks
Mark

**Comment by yaminncco** [comment]
the component doesn't check for query at the moment, what you can do is to use param instead

```
{
    href: '/domain/1',
    title: 'product1',
    icon: 'fa fa-file-alt'
},
{
    href: '/domain/2',
    title: 'product2',
    icon: 'fa fa-file-alt'
}

```
```
{
    path: '/domain/:pid',
    component: require('./pages/Dashbroad.vue').default,
    meta: {
        requiredUser: ['default']
    }
}
```

**Comment by darklord1990** [comment]
As you said, it just applied for 1 param. But  I need to use multi params. How can I do it ?

`href: '/domain/?pid=2&gid=0'`

**Comment by yaminncco** [comment]
new update should fix your issue

**Comment by darklord1990** [comment]
Thanks @yaminncco .

**Comment by darklord1990** [comment]
Hi @yaminncco ,  is this issue already fixed at update 3.7.1 ?

**Comment by darklord1990** [comment]
Yes, It works. Thanks for your quickly update.
